elastic/kibana
# [Index Management] Remove freeze index action and api

Fixes https://github.com/elastic/kibana/issues/115837

### Summary

Since the freeze API is being removed from ES, we should remove the action from the Index Management UI and API. But we  still want to allow users to unfreeze indices, to give them an escape hatch for unfreezing existing frozen indices in their cluster.

**How to test**
1. Start up elasticsearch, `yarn es snapshot --license=trial`
2. Start up Kibana, `yarn start`
3. Go to Console and create a new index and freeze it:
```
PUT /test_index
POST /test_index/_freeze
```
4. Navigate to `Stack Management` -> `Index Management`
5. Verify that the newly created `test_index` is marked as frozen
6. Verify that clicking on it, opening the `Manage` popover shows the `Unfreeze index` option
7. Verify that clicking the `Unfreeze index` option works as expected
8. Verify that Opening the `Manage` popover again doesn't show the `Freeze index` action
